src/chains/chains.controller.ts

```diff
@@ -1,9 +1,12 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, VERSION_NEUTRAL } from '@nestjs/common';
 import { ChainsService } from './chains.service';
 import { Chain } from './entities/chain.entity';
 import { Page } from './entities/page.entity';
 
-@Controller('chains')
+@Controller({
+  path: 'chains',
+  version: VERSION_NEUTRAL,
```

Comment:
This will change because in production it's under `v1` too. You can either change it here or leave this one as is and we tackle in a different one.

Reply:
Oh, right, you used the root just for testing I guess, I'll move it to the `/v1` path in this PR. At least we know this is working fine with no version-prefixed endpoints 👍

Reply:
Yeah that's right! Thanks Hector!